Documentation/deploy-master.md

```diff
@@ -668,6 +668,14 @@ kube-controller-$node
 kube-proxy-$node
 ```
 
+### Configure Calico
+
+To start Calico services :
+
+```
+docker run --rm --net=host -v /srv/kubernetes/manifests:/host/manifests quay.io/coreos/hyperkube:v1.5.1_coreos.0 /hyperkube kubectl apply -f /host/manifests/calico.yaml
```

Comment:
> Running kubectl not in a container by downloading the binary.

This would mean either duplicating the kubectl set up instructions, or moving them up from the 3rd step. Does it make sense to add this Calico part onto this page: https://github.com/coreos/coreos-kubernetes/blob/master/Documentation/deploy-addons.md

It's kind of like an addon, so I can see that working from a "deploy cluster then addons" mental flow.